Extensions/FileSystem/JsExtension.js

```diff
@@ -59,6 +59,77 @@ module.exports = {
       )
       .setFunctionName("gdjs.fileSystem.makeDirectory");
 
+    extension
+      .addAction(
+        "SaveStringToFile",
+        t("Save a text into a file"),
+        t("Save a text into a file."),
+        t("Save _PARAM0_ into file _PARAM1_"),
+        t("Filesystem/Windows, Linux, MacOS"),
+        "JsPlatform/Extensions/filesystem_save_file24.png",
+        "JsPlatform/Extensions/filesystem_save_file32.png"
+      )
+      .addParameter("string", t("String (text)"), "", false)
+      .addParameter("string", t("Save path"), "", false)
+      .getCodeExtraInformation()
+      .setIncludeFile(
+        "Extensions/FileSystem/filesystemtools.js"
+      )
+      .setFunctionName("gdjs.fileSystem.saveStringToFile");
+
+    extension
+      .addAction(
+        "LoadStringFromFileAsync",
+        t("Load a text from a file (Async)"),
+        t("Load a text from a file asynchronously. Use this for large files to not delay game execution. (The content of the file will not be available immediately)"),
```

Comment:
As below, i would say "avoid any lag or freeze during game execution."
Also, better write sentence in a positive form explaining how to get the result:

`Load a text from a file asynchronously. Use this for large files to avoid any lag or freeze during game execution. The content of the file will be available in the scene variable after a small delay (usually a few milliseconds).`